providers/record.rb

```diff
@@ -79,7 +79,7 @@ def record_value_or_alias_attributes
 end
 
 action :create do
-  require 'fog'
+  require 'fog/aws/dns'
```

Comment:
You are right, latest versions break with that change:

```
[5] pry(main)> require 'fog/aws/dns'
NameError: uninitialized constant Fog::Service
from /Users/joseluis/code/platform/tmp/vendor/ruby/1.9.1/gems/fog-aws-0.4.0/lib/fog/aws/dns.rb:3:in `<module:DNS>'
```

I'm going to push the fix right now, thanks.